practical-penguins/trivia_tavern/trivia_runner/templates/activequiz_question.html

```diff
@@ -0,0 +1,62 @@
+{% extends "base.html" %}
+{% block content %}
+{% load static %}
+<script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+<script src="{% static 'countdowntimer.js' %}"></script>
+<script>
+
+  window.onload = function() {
+    //hard coded to thirty seconds. Change this to the variable the host sets
+    var display = document.querySelector("#time"),
+        timer = new CountDownTimer(30);
```

Comment:
It looks like the feature of custom timer duration didn't make it in to the final version. But it's definitely good to see that it was considered as something to expand upon.